### PR TITLE
More lenient scar installation specifications

### PR DESCRIPTION
**Pull request by fgypas**
- More lenient scar installation specifications
- Split scAR.yml to scAR-gpu.yaml and scAR-cpu.yaml
- Update documentation

### CODE REVIEW

File: scAR-cpu.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- nvidia
+- pytorch
+- conda-forge
+dependencies:
+- conda-forge::python>=3.8.6
+- nvidia::cudatoolkit>=11.1
+- pytorch::pytorch>=1.10.0
+- pytorch::tcorchvision>=0.9.0
+- pytorch::torchaudio>=0.8.0
+- pytorch-mutex=*=cpu
```
Comment (fgypas) [inline scAR-cpu.yml:12]: `pytorch-mutex` is a conda metapackage that you can use to instruct conda to install the cpu or the cuda (gpu) version of the package.

File: scAR-cpu.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+name: scAR
+channels:
+- nvidia
+- pytorch
+- conda-forge
+dependencies:
+- conda-forge::python>=3.8.6
```
Comment (fgypas) [inline scAR-cpu.yml:7]: I am not sure if this is a good practice, but I specified from where to fetch the dependencies

File: scAR-gpu.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- nvidia
+- pytorch
+- conda-forge
+dependencies:
+- conda-forge::python>=3.8.6
+- nvidia::cudatoolkit>=11.1
+- pytorch::pytorch>=1.10.0
+- pytorch::tcorchvision>=0.9.0
+- pytorch::torchaudio>=0.8.0
+- pytorch-mutex=*=cuda
```
Comment (fgypas) [inline scAR-gpu.yml:12]: gpu version

File: .github/workflows/python-conda-build.yaml [hunk 4]
```diff
@@ -19,7 +19,7 @@ jobs:
           echo $CONDA/bin >> $GITHUB_PATH
       - name: Install dependencies
         run: |
-          conda env create -f scAR.yml
+          conda env create -f scAR-cpu.yml
```
Comment (fgypas) [inline .github/workflows/python-conda-build.yaml:22]: Update action to use the cpu version of the environment.

**Comment by fgypas** [comment]
Also the GPU version of the test works locally.

**Review by CaibinSh — APPROVED** [review]

**Review by Tobias-Ternent — APPROVED** [review]

**Pull request merged**